Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py

```diff
@@ -0,0 +1,146 @@
+        report_content = 'This is a dumped content of the article. Use the link under Publications field to read ' \
+                         'the full article. \n\n'
+        soup = BeautifulSoup(response_url.text, "html.parser")
+        for tag in soup.find_all():
+            if tag.name in HTML_TAGS:
+                for string in tag.stripped_strings:
+                    report_content += ' ' + string
+        if len(report_content.encode('utf-8')) > self.content_max_size:  # Ensure report_content does not exceed the
+            # indicator size limit (~50KB)
+            report_content = report_content.encode('utf-8')[:self.content_max_size].decode('utf-8')
+        return report_content
+
+    def fetch_indicators(self):
+        self.feed_data = feedparser.parse(self.feed_response.text)
+        self.parsed_indicators = self.create_indicators_from_response()
+
+
+def get_indicators(client: Client, args: dict) -> CommandResults:
+    limit = int(args.get('limit', 10))
+    parsed_indicators = client.parsed_indicators[:limit]
+    parsed_for_hr = [{'Title': indicator.get('value'),
+                      'Link': indicator.get('fields').get('publications')[0].get('link'),
+                      'Type': indicator.get('type')}
+                     for indicator in parsed_indicators]
+    headers = ['Title', 'Link', 'Type']
+    hr_ = tableToMarkdown(name='RSS Feed:', t=parsed_for_hr, headers=headers)
+    return CommandResults(
+        readable_output=hr_,
+        raw_response=client.feed_data
+    )
+
+
+def main():
+    params = demisto.params()
+    server_url = params.get('server_url')
+    command = demisto.command()
+    demisto.info(f'Command being called is {command}')
+    try:
+        client = Client(server_url=server_url,
+                        use_ssl=not params.get('insecure', False),
+                        proxy=params.get('proxy'),
+                        feed_tags=argToList(params.get('feedTags')),
+                        tlp_color=params.get('tlp_color'),
+                        content_max_size=int(params.get('max_size', '45')))
+
+        if command == 'test-module':
+            if 'html' in client.feed_response.headers['content-type']:
+                raise DemistoException(f'{server_url} is not rss feed url. Try look for a url containing \'feed\' '
+                                       f'prefix or suffix.')
+            else:
+                return_results("no ok")
+                # return_results("ok")
+
+        elif command == 'rss-get-indicators':
+            client.fetch_indicators()
+            return_results(get_indicators(client, demisto.args()))
+
+        elif command == 'fetch-indicators':
+            client.fetch_indicators()
+            for iter_ in batch(client.parsed_indicators, batch_size=2000):
+                demisto.createIndicators(iter_)
+        else:
+            raise NotImplementedError(f'Command {command} is not implemented.')
```

Comment:
please either space out all clauses of the if/else block, or space out neither of them.
```suggestion
        if command == 'test-module':
            if 'html' in client.feed_response.headers['content-type']:
                raise DemistoException(f'{server_url} is not rss feed url. Try look for a url containing \'feed\' '
                                       f'prefix or suffix.')
            else:
                return_results("no ok")
                # return_results("ok")
        elif command == 'rss-get-indicators':
            client.fetch_indicators()
            return_results(get_indicators(client, demisto.args()))
        elif command == 'fetch-indicators':
            client.fetch_indicators()
            for iter_ in batch(client.parsed_indicators, batch_size=2000):
                demisto.createIndicators(iter_)
        else:
            raise NotImplementedError(f'Command {command} is not implemented.')
```